ElectronClient/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -500,8 +502,9 @@ const TinyMCE = (props:NoteBodyEditorProps, ref:any) => {
 				target_list: false,
 				table_resize_bars: false,
 				language: ['en_US', 'en_GB'].includes(language) ? undefined : language,
-				toolbar: 'bold italic | link joplinInlineCode joplinCodeBlock joplinAttach | numlist bullist joplinChecklist | h1 h2 h3 hr blockquote table joplinInsertDateTime',
+				toolbar: 'bold italic | link joplinInlineCode joplinCodeBlock joplinAttach | numlist bullist joplinChecklist | h1 h2 h3 hr blockquote table joplinInsertDateTime', // TODO: Look into spellchecker toolbar plugin
```

Comment:
Is this working for you? As last time I tried it didn't work for me, but maybe I've configured it differently.

Reply:
For the toolbar? Yeah I could get the spellcheck button to appear but it didn't do anything on click, it just says "The spelling service was not found: (undefined)". It still works within the editor without the toolbar plugin though.